sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/_package/model_configuration.py

```diff
@@ -12,7 +12,7 @@
 class ModelConfiguration:
     """ModelConfiguration.
 
-    :param mode: The mode of the model.
+    :param mode: The mode of the model. Possible values include: "Copy", "Download"..
```

Comment:
Do you need to change the schema also, looks like "Copy" is not allowed. Also did you check if this is supported in the API, the current _restclient RestModelConfiguration, does not have "Copy" documented, doyou need to check the latest swagger?